etc/kayobe/pulp.yml

```diff
@@ -128,6 +129,69 @@ stackhpc_pulp_repository_deb_repos:
     state: present
     required: "{{ stackhpc_pulp_sync_ubuntu_focal | bool }}"
 
+  # Base Ubuntu Jammy repositories
+  - name: Ubuntu jammy
+    url: "{{ stackhpc_release_pulp_content_url }}/ubuntu/jammy/{{ stackhpc_pulp_repo_ubuntu_jammy_version }}"
+    remote_username: "{{ stackhpc_release_pulp_username }}"
+    remote_password: "{{ stackhpc_release_pulp_password }}"
+    client_cert: ""
+    client_key: ""
+    policy: on_demand
+    proxy_url: "{{ pulp_proxy_url }}"
+    architectures: amd64
+    components: main restricted universe multiverse
+    distributions: jammy jammy-updates jammy-backports
+    mirror: true
+    state: present
+    required: "{{ stackhpc_pulp_sync_ubuntu_jammy | bool }}"
+
+  - name: Ubuntu jammy security
+    url: "{{ stackhpc_release_pulp_content_url }}/ubuntu/jammy-security/{{ stackhpc_pulp_repo_ubuntu_jammy_security_version }}"
+    remote_username: "{{ stackhpc_release_pulp_username }}"
+    remote_password: "{{ stackhpc_release_pulp_password }}"
+    client_cert: ""
+    client_key: ""
+    policy: on_demand
+    proxy_url: "{{ pulp_proxy_url }}"
+    architectures: amd64
+    components: main restricted universe multiverse
+    distributions: jammy-security
+    mirror: true
+    state: present
+    required: "{{ stackhpc_pulp_sync_ubuntu_jammy | bool }}"
+
+  # Ubuntu Cloud Archive (UCA) Jammy repositories
+  - name: Ubuntu Cloud Archive
+    url: "{{ stackhpc_release_pulp_content_url }}/ubuntu-cloud-archive/{{ stackhpc_pulp_repo_ubuntu_cloud_archive_version }}"
+    remote_username: "{{ stackhpc_release_pulp_username }}"
+    remote_password: "{{ stackhpc_release_pulp_password }}"
+    client_cert: ""
+    client_key: ""
+    policy: on_demand
+    proxy_url: "{{ pulp_proxy_url }}"
+    architectures: amd64
+    components: main
+    distributions: "jammy-updates/{{ openstack_release }}"
+    mirror: true
+    state: present
+    required: "{{ stackhpc_pulp_sync_ubuntu_jammy | bool }}"
+
+  # Third-party repositories for Jammy
+  - name: Docker CE for Ubuntu
```

Comment:
This has the same name as the focal one, so ansible modules will get confused and reuse the same repository.

Can we use the same definition for both, with multiple `distributions`?